src/drivers/actuators/modal_io/modal_io.cpp

```diff
@@ -1169,9 +1125,10 @@ bool ModalIo::updateOutputs(bool stop_motors, uint16_t outputs[MAX_ACTUATORS],
 		actuator_outputs.timestamp = hrt_absolute_time();
 
 		_outputs_debug_pub.publish(actuator_outputs);
```

Comment:
How is this different than the regular `actuator_outputs` publication?

Reply:
It is the actual values sent to the ESC. It takes what is provided by the mixer module (and published as actuator_outputs) and possibly reverses direction based on parameter settings or turtle mode. So can be helpful for ESC driver debug.